signer_install.md

```diff
@@ -0,0 +1,149 @@
+# Kritis Signer
```

Comment:
nit: can we link to this from the main README.md so users can find it more easily?

Reply:
Done.